Bottom: 2737e70c64c9617bb741fff20192947019c4f7d9
Top:    03e0d857be2abeca4aaf50a1c372dc983b1332dc
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-03-05 17:18:11 -0800

librdmacm: Release 1.0.17


---

diff --git a/configure.in b/configure.in
index d4434c2..35d79dd 100644
--- a/configure.in
+++ b/configure.in
@@ -1,12 +1,12 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
-AC_INIT(librdmacm, 1.0.16, linux-rdma@vger.kernel.org)
+AC_INIT(librdmacm, 1.0.17, linux-rdma@vger.kernel.org)
 AC_CONFIG_SRCDIR([src/cma.c])
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(config)
-AM_CONFIG_HEADER(config.h)
-AM_INIT_AUTOMAKE(librdmacm, 1.0.16)
+AC_CONFIG_HEADERS(config.h)
+AM_INIT_AUTOMAKE(librdmacm, 1.0.17)
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 AM_PROG_LIBTOOL
diff --git a/librdmacm.spec.in b/librdmacm.spec.in
index 575ce84..7ad6c0c 100644
--- a/librdmacm.spec.in
+++ b/librdmacm.spec.in
@@ -1,7 +1,7 @@
 %define ver @VERSION@
 
 Name: librdmacm
-Version: 1.0.16
+Version: 1.0.17
 Release: 1%{?dist}
 Summary: Userspace RDMA Connection Manager
